src/components/navLinks.tsx

```diff
@@ -0,0 +1,19 @@
+export const navLinks = [
+  { name: "Home", path: "/" },
+  {
+    name: "Event",
```

Comment:
Got it, the above reload is happening because the pages do not exist yet! Not a problem then!